rest/rest.go

```diff
@@ -56,17 +57,9 @@ type Server struct {
 	// ServeUnix sets whether the resources of this endpoint should also be served over the unix socket.
 	ServeUnix bool
 
-	// Protocol is the server protocol.
-	// Example: https
-	Protocol string
-
-	// Address is the server listen address.
-	// Example: 127.0.0.1:9000
-	Address types.AddrPort
-
-	// Certificate is used for setting up TLS on the server.
-	// x509 PEM Certificate
-	Certificate *shared.CertInfo
+	// DedicatedCertificate sets whether the additional listener should use its own self signed certificate.
+	// If false the default cluster certificate from the core API is used.
+	DedicatedCertificate bool
```

Comment:
Could you please add to the comment here that the certificate will be searched for in the `/certificates` directory of the state directory, with the name given to the server when supplied to `microcluster.Start`